plugin/nvimgdb.vim

```diff
@@ -9,13 +9,24 @@ function! s:Spawn(backend, proxy_cmd, client_cmd)
   call luaeval("require'nvimgdb'.new(_A[1], _A[2], _A[3])", [a:backend, a:proxy_cmd, cmd])
 endfunction
 
-command! -nargs=1 -complete=shellcmd GdbStart call s:Spawn('gdb', 'gdb_wrap.sh', <q-args>)
+command! -nargs=1 -complete=customlist,ExecsCompletion GdbStart call s:Spawn('gdb', 'gdb_wrap.sh', <q-args>)
 command! -nargs=1 -complete=shellcmd GdbStartLLDB call s:Spawn('lldb', 'lldb_wrap.sh', <q-args>)
 command! -nargs=1 -complete=shellcmd GdbStartPDB call s:Spawn('pdb', 'pdb_proxy.py', <q-args>)
 command! -nargs=1 -complete=shellcmd GdbStartBashDB call s:Spawn('bashdb', 'bashdb_proxy.py', <q-args>)
 
+let g:use_find_executables=1
+let g:find_executables_base_dir=g:cmake_build_dir
+let g:use_cmake_to_find_executables=1
+function ExecsCompletion(ArgLead, CmdLine, CursorPos)
+  let found_executables = g:use_find_executables ? 
+        \systemlist("find " . g:find_executables_base_dir . " -type f -executable") : []
+  let cmake_executables = g:use_cmake_to_find_executables ? 
+        \ExecutableOfBuffer() : []
+  return extend(cmake_executables, found_executables)
+endfunction
+
 if !exists('g:nvimgdb_disable_start_keymaps') || !g:nvimgdb_disable_start_keymaps
-  nnoremap <leader>dd :GdbStart gdb -q a.out
+  nnoremap <leader>dd :GdbStart gdb -q 
```

Comment:
By the way, this change breaks automatic tests already because most of them rely on `a.out` being debugged by default. I don't know why this wasn't noticed by the git workflow runs.

Reply:
Yes, the tests failed but the checks didn't notice that and showed "success". Must be a bug in the workflow.

Reply:
I haven't run any of the tests. I'll try to see what I can do about that. Maybe add my own...

Reply:
There're a couple of options: leave `a.out` as default, fix the tests to make sure the executable `a.out` is selected. Or maybe better to use your code to select a random target found around?
You can see how to run the test suite in the `.github` directory, by the way. Basically, it's just `cd test; ./prerequisites.sh; ./run -vv ..`

Reply:
I added `a.out` to the gdb a lldb tests. I can't find a `bashdb` for bash 5.1 so I haven't run the tests on my PC yet. Maybe I'll try podman or something